src/WebJobs.Extensions.DocumentDB/Trigger/DocumentDBTriggerAttribute.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Extensions
+{
+    using System;
+    using Microsoft.Azure.WebJobs.Extensions.DocumentDB;
+
+    /// <summary>
+    /// Defines the [DocumentDBTrigger] attribute
+    /// </summary>
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1019:DefineAccessorsForAttributeArguments")]
+    [AttributeUsage(AttributeTargets.Parameter)]
+    public sealed class DocumentDBTriggerAttribute : Attribute
```

Comment:
Just curious, without the multiple constructors, how would you specify the other parameters in a Functions signature?

For example:

`public static void CosmosSample([DocumentDBTrigger("connectionString", "db", "collection", "leaseConnectionString", "db", "collection")] JArray docs)`

How would I pass the lease data if I take away the constructor that receives it?